ChangeLog

```diff
@@ -83,6 +171,33 @@
 - Relaxed AI requirement for hunters
 - AI error is more informative when building is missing the
   "collects_ware_from_map" AI hint
+- Replace some asserts in the AI with more informative Exceptions for tribe/world designers. (#3524)
+- Limit ware/worker preciousness for the AI. (#3530): Warn if ware/worker preciousness is a bit high and throw an exception if it's over 200.
+- AI supports seafaring (#3528). This introduces new ai_hint "supports_seafaring", that will be used to boost building score for particular building when on seafaring map.
+- AI fixes (#3602)
+  - fixed crude percent to prevent overflow
+  - AI worker management
+  - implemented a worker management for AI Productionsites and mines.
+  - added a malus for unoccupied sites to the build decisions.
+  - some small improvements for upgradehandling of buildings
+- About 100 rounds of training
```

Comment:
all training rounds should be handled by one line I wouln't even write down the number of rounds. 
Proposal: Use "further training of AI as placeholder